sawtooth-daml-rpc/src/main/java/com/blockchaintp/sawtooth/daml/rpc/SawtoothReadService.java

```diff
@@ -121,14 +119,14 @@ private TimeModel parseTimeModel(final ByteString data) throws InvalidProtocolBu
     TimeModel tm;
     if (data == null) {
       LOGGER.info("No time model set on chain using defaults");
-      tm = new TimeModel(Duration.ofSeconds(1), Duration.ofSeconds(DEFAULT_MAX_CLOCK_SKEW),
-          Duration.ofSeconds(DEFAULT_MAX_TTL));
+      tm = TimeModel.apply(Duration.ofSeconds(1), Duration.ofSeconds(DEFAULT_MAX_CLOCK_SKEW),
+          Duration.ofSeconds(DEFAULT_MAX_TTL)).get();
```

Comment:
Is this really the way it has to be done now? seems odd for a non-async/future-y kind of thing.

Reply:
same deal on public Object vs private class.  the public object only has the apply method that returns a Try --> will also see why this was changed and whether there is a nicer way to interface

Reply:
i went back to the old syntax and although the IDE complains it seems to compile